src/common/ratings.c

```diff
@@ -59,10 +111,26 @@ void dt_ratings_apply_to_selection(int rating)
 
     /* for each selected image update rating */
     sqlite3_stmt *stmt;
+    bool first = true;
```

Comment:
do not use bool it breaks some compilers, use gboolean.

Reply:
Done.